ports/scintilla/vcpkg.json

```
@@ -3,7 +3,7 @@
   "version": "5.5.1",
   "description": "A free source code editing component for Win32, GTK+, and OS X",
   "homepage": "https://www.scintilla.org/",
-  "supports": "!(uwp | linux | osx)",
+  "supports": "!(uwp | linux | osx | android)",
```

Well, upstream does support osx and linux, but the port doesn't.
As long as there is only msbuild here:
```suggestion
  "supports": "windows & !uwp & !mingw",
```

---

Not resolved. It also fails to build for ios, freebsd, openbsd, wasm... It just not tested in CI.